src/components/VoteScreen.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import CandidateStatus from './CandidateStatus.js';
+import LoginInput from './LoginInput';
+import { useCookies } from 'react-cookie';
+
+const candiadateURL='http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/candidates'
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
```

Comment:
배열이면 기본값을 빈 배열로 해주는게 좋아보여용

```suggestion
  const [candidates, setCandidates] = useState([]);
```